scripts/process_crashes.sh

```diff
@@ -46,6 +46,7 @@ mkdir "${DATADIR}" || echo "${DATADIR} already exists."
 
 # Add crash ids to queue
 ./socorro-cmd add_crashid_to_queue socorro.normal $@
+# ./socorro-cmd pubsub publish $@
```

Comment:
When we switch to Pub/Sub, we can uncomment this line and comment out the above line.